Gemfile

```diff
@@ -65,6 +65,7 @@ group :test, :development do
   gem 'rack-test', :require => 'rack/test'
   gem 'simplecov', :require => false
   gem 'pry-byebug'
+  gem 'pry-doc'
```

Comment:
Too trivial for an additional PR, unless we really need to discuss it and review it etc?